snippets/zoe/contracts/test-zoe-01-hello.js

```diff
@@ -0,0 +1,17 @@
+// #region test-01-hello
+// #region test-imports
+import '@endo/init';
+import { E } from '@endo/far';
+// eslint-disable-next-line import/no-unresolved -- https://github.com/avajs/ava/issues/2951
+import test from 'ava';
+// #endregion test-imports
+// #region test1
+import { start } from '../src/01-hello.js';
+
+test('contract greets by name', async t => {
+  const { publicFacet } = start();
+  const actual = await E(publicFacet).greet('Bob');
+  t.is(actual, 'Hello, Bob!');
+});
```

Comment:
TECHNICAL:
I don't understand why we should maintain all this code in a 2 separate files.

Reply:
I think it is neat and structured this way and rather easier to maintain.

Reply:
I'd like to understand what prompted moving the code in the 1st place.
But I guess I don't feel strongly about it.

Reply:
Well, the file contained tests for unrelated contracts.